src/main/java/org/jenkinsci/plugins/workflow/job/console/NewNodeConsoleNote.java

```diff
@@ -131,7 +131,7 @@ public static StringBuilder startTagFor(@NonNull WorkflowRun context, @NonNull S
                     }
                 }
             } catch (IOException x) {
-                Logger.getLogger(NewNodeConsoleNote.class.getName()).log(Level.WARNING, null, x);
+                LOGGER.log(Level.FINE, "unloadable " + context, x);
```

Comment:
For what it's worth I think this overload exists:

```suggestion
                LOGGER.log(Level.FINE, x, () -> "unloadable " + context);
```